plugins/wpe-headless/includes/rest/callbacks.php

```diff
@@ -18,24 +18,26 @@
  *
  * @link https://www.wpgraphql.com/filters/graphql_authentication_errors/
  *
+ * @param bool $authentication_errors Whether there are authentication errors with the request.
+ *
  * @return bool True if the Authorization header exists and is invalid, false otherwise.
  */
-function wpe_headless_rest_validate_access_token() {
+function wpe_headless_rest_validate_access_token( $authentication_errors ) {
```

Comment:
I wonder if we can improve the name of this function. With a return value of `true` meaning something like "there are authentication errors", it doesn't quite fit the name. I would expect a function named `wpe_headless_rest_validate_access_token` to return `true` if the token is valid.

Reply:
Maybe something like `wpe_headless_graphql_authentication_errors` or similar?

Reply:
Dang, good Q. It is somewhat awkward.

Perhaps something like `wpe_headless_graphql_has_auth_errors`? I'm a fan of predicates having `is` or `has` in their name.